network/load-balancer/reference-content/acls.mdx

```diff
@@ -0,0 +1,116 @@
+
+If traffic reaches the end of the list of ACLs without any actions being applied to it, it is allowed to pass to the backend. If you want to avoid this, create an unconditional `Deny`  ACL and place it at the end of your list of conditional `Allow` ACLs.
+
+## What conditions can be set for ACLs?
+
+For ACLs which have **Enable condition** activated, you must specify a **condition type** and a **filter**.
+
+Condition types are:
+  - **IF MATCH**: The ACL applies its action to all traffic that matches the given filters.
+  - **IF NO MATCH**: The ACL applies its action to all traffic that does **not** match the given filters .
+
+Filter types are:
+  - **IP filter**: This filter type is available for all frontends. You can filter for one or many source IPv4 or IPv6 addresses, or IP blocks.
+  - **HTTP filter**: This filter type is only available for frontends attached to HTTP backends. You can filter for one or many values at either the **beginning** or **end** of the traffic's HTTP request path, or for a **regex** pattern across the entire HTTP request path. Alternatively, you can set an HTTP **header** to filter, and the **values** to filter for within that header.
+
+You can only choose one filter type per ACL, though you can filter for many values within that filter type (multiple IP addresses in teh case of an IP filter, or multiple HTTP values within a specific HTTP filter type).
+
+## Using ACLs to filter traffic's access to the backend
+
+Below, we show how to achieve a number of different filtering objectives with ACLs
+
+### Only allow traffic from certain IPs
+
+In this case, we only want traffic from source IPs `51.158.99.227` and `151.115.78.39` to be allowed to pass to the backend.
+
+#### Option 1
+Construct two ACLs:
+- **ACL Priority 1**: `Allow` - `Enable condition` - `IF MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will allow traffic from the two specified IPs to pass directly to the backend. All traffic not from these IPs will pass to the next ACL.
+- **ACL Priority 2**: `Deny` - `Disable condition`. This ACL will deny access to all remaining traffic.
+
+#### Option 2
+Construct one ACL:
+- **ACL Priority 1**: `Deny` - `Enable condition` - `IF NOT MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will reject any traffic **not** from the two specified IPs. Traffic from the specified IPs will, by default, pass to the backend since there are no further ACLs in the list.
+
+### Reject traffic from certain IPs, and with certain HTTP header values.
+
+In this case, we want to reject traffic from source IPs `51.158.99.227` and `151.115.78.39`, and also reject traffic with either of these IP addresses in the `X-Forwarded-For` HTTP header. All other traffic should be allowed access.
+
+Construct two ACLs:
```

Comment:
I think the example is awesome, great job!
One thing that came to my mind when I was reading it: why not doing a single ACL. Then I plugged my brain and realized that when you are using 1 IP filter and 1 HTTP filter in an ACL then they they should both be true (logical AND between the 2), so a single ACL wouldn't work in this case unless we use an IF NO MATCH type.
I am then wondering if we should explain how multiple filters work together before jumping to the examples ?